source/features/remove-zero-packages.tsx

```diff
@@ -0,0 +1,22 @@
+import select from 'select-dom';
+import features from '../libs/features';
+import {getRepoURL} from '../libs/utils';
+
+function init(): void {
+	const packageElem: HTMLElement | null = select(`.numbers-summary a[href='/${getRepoURL()}/packages']`);
+	if (packageElem && packageElem.innerText.trim() === '0 packages') {
+		const parentElem = packageElem.parentElement;
+		parentElem!.remove();
+	}
+}
+
+features.add({
+	id: __featureName__,
+	description: 'Hides the packages tab, when no packages exist',
+	screenshot: false,
```

Comment:
As this is a visual change, this feature should have a screenshot attached to it.

Follow the same code changes of #2222, mention the feature in readme under repositories section, copying the after and before text from [this](https://user-images.githubusercontent.com/37769974/59083449-0ef88f80-8915-11e9-8296-68af1ddcf191.png) screenshot.